test/toolchainconfig_assertions.go

```diff
@@ -0,0 +1,71 @@
+package test
+
+import (
+	"context"
+
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/api/v1alpha1"
+	"github.com/codeready-toolchain/toolchain-common/pkg/test"
+	"github.com/stretchr/testify/require"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type ToolchainConfigAssertion struct {
+	config         *toolchainv1alpha1.ToolchainConfig
+	client         client.Client
+	namespacedName types.NamespacedName
+	t              test.T
+}
```

Comment:
Wouldn't it make sense to move the methods to toolchain-common repo? I guess that they could be useful also in e2e tests. WDYT?

Reply:
I thought we'd focus on checking only the status and actual config behaviour in e2e but it doesn't hurt to have it in toolchain-common so I'll add it there.

Reply:
Done https://github.com/codeready-toolchain/host-operator/pull/461/commits/709b826b8247c6b3232684d9627ea7f3e600723b

And here's the PR for toolchain-common: https://github.com/codeready-toolchain/toolchain-common/pull/188